openhsi/data.py

```diff
@@ -277,15 +277,16 @@ def rad2ref_6SV(self:CameraProperties, x:Array['λ,x',np.float32]) -> Array['λ,
 @patch
 def set_processing_lvl(self:CameraProperties, lvl:int = 2, custom_tfms:List[Callable[[np.ndarray],np.ndarray]] = None):
     """Define the output `lvl` of the transform pipeline.
-    0 : raw digital numbers cropped to useable sensor area
-    1 : case 0 + fast smile correction
-    2 : case 1 + fast binning (default)
-    3 : case 1 + slow binning
-    4 : case 2 + conversion to radiance in units of uW/cm^2/sr/nm
-    5 : case 4 except radiance conversion moved to 2nd step
-    6 : case 4 + conversion to reflectance
-    7 : smile corrected and binned -> radiance
-    8 : case 7 + converted to reflectance
+    -1: do not apply any transforms,
+    0 : raw digital numbers cropped to useable sensor area,
+    1 : case 0 + fast smile correction,
+    2 : case 1 + fast binning (default),
+    3 : case 1 + slow binning,
+    4 : case 2 + conversion to radiance in units of uW/cm^2/sr/nm,
+    5 : case 4 except radiance conversion moved to 2nd step,
```

Comment:
current structure of code means we can't do radiance conversion before smile and wave binning, as radiance conversion uses wave scale.  This needs to rethought down the road, but right now that mode will fail, so should be removed/disabled.

Reply:
I wanted radiance conversion to be done first thing if you wanted. The current structure is a limitation of what integrating sphere data I had on hand (so already smile corrected and binned). This will take some experimenting to update.

Reply:
Yeah, I think right cause it uses the rad cal from the sfit, it does not work. We should mark this as an issue too address later.

I can share some raw data and cal data from  one of the robotx OpenHSIs you can test with as well.

Reply:
An integrating sphere dataset (I only really need the hypercube shape and named coordinates) will be very helpful to resolve this. I can test on the FLIR camera. We'll have to fully calibrate and focus the FLIR camera later - and grab a grub screw because the lens tube keeps rotating on me.

Reply:
yes, I'll share some raw data and cal data you can use to test with capture.OpenHSI class for testing before we can calibrate the FLIR sensor.